portafly/src/i18n/locales/en/account_settings/personal/personal_details.yml

```diff
@@ -1,5 +1,63 @@
 ---
+# Account settings > Personal > Personal details
+
 page_title: Personal details | Portafly
-page_title_description: Page title of the subsection for Personal details
+page_title_description: Page title of the subsection for Personal details.
+
+# Top area
+
 body_title: Personal details
-body_title_description: Body title of the page for Personal details
+body_title_description: Body title of the page for Personal details.
+
+# Main area
+
+# Tabs
+
+user_data_title: User data
+user_data_description: This is the title of the tab containing the user's personal information.
+change_password_title: Change password
+change_password_description: This is the title of the tab containing the user's current password field and new password field.
+
+# User data form
+
+username_label: Username
+username_label_description: Label for the field that holds the current username.
+username_validation: "{{username}} is too short. Use a minimum of 3 characters. Do not use spaces. You can include letters, numbers, and the following special characters . - _ @ +"
```

Comment:
@dfennessy In this example, {{username}} is a placeholder. Does this answer your question?

Reply:
@porueesq isn't {{username}} a variable?
(just want to make sure we're not confusing variables with placeholder text)

Reply:
Yes, it is a variable. In this case, acting as a placeholder for the user name, though technically it's a variable.